custom_components/octopus_energy/heat_pump/__init__.py

```diff
@@ -344,6 +344,32 @@ def mock_heat_pump_status_and_configuration():
           }
         }
       ]
+    },
+    "octoHeatPumpLifetimePerformance": {
```

Comment:
suggestion: These values should be randomised (see telemetry). This is useful for me when turning on mocking when testing this feature that I don't have live access to.